orttraining/orttraining/test/python/orttraining_test_pytorch_trainer_frontend.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import torch
+
+from onnxruntime.capi.training.optim import lr_scheduler
+from onnxruntime.capi.training.amp import loss_scaler
+from onnxruntime.capi.training import pytorch_trainer_options as pt_options
```

Comment:
Is it necessary to keep everything training related under `capi`? For frontend part can we move things to just `onnxruntime.training`?

Reply:
That's a great point and slightly related to https://github.com/microsoft/onnxruntime/pull/3870

Classes names will probably change due to some ongoing discussions, so we should come back to this soon.
This implementation keeps back compatibility until a decision is made

Reply:
Good point. I agree that we should consider moving out of `capi` namespace soon.